pkg/volume/plugins.go

```diff
@@ -743,7 +743,8 @@ func (pm *VolumePluginMgr) refreshProbedPlugins() {
 	events, err := pm.prober.Probe()
 	if err != nil {
 		klog.Errorf("Error dynamically probing plugins: %s", err)
-		return // Use cached plugins upon failure.
+		//skipping events will lead to fail to add all health plugins driver once one plugin file is not ready, always keeping consume health events.
```

Comment:
I understand the idea of this change but unfortunately I don't think that ignoring the error is the solution. The contract `Probe()` provides is explicitly: `// If an error occurs, events are undefined.` so we cannot rely on processing events when there is an error. The solution may be more involved to do separate probing for each plugin.

Reply:
I could add an check to avoid undefined events, in this case, only health events are added into events array, it should be continue to register these plugin.

Reply:
sure, could you implement that and so I can review that solution. Currently I don't know what that would look like

Reply:
@davidz627 see my impl in following commit and feel free to comment!